tests/helpers/vscode.js

```
@@ -17,6 +17,14 @@ class Uri {
     return new Uri(value, scheme, fsPath);
   }
 
+  static joinPath(base, ...segments) {
+    const joined = [base.fsPath, ...segments]
+      .map((segment) => String(segment).replace(/^[/\\]+|[/\\]+$/g, ''))
```

ESLint

[error] Prefer `String#replaceAll()` over `String#replace()`.

(unicorn/prefer-string-replace-all)